notebooks/vertex_genai/labs/prompt_engineering.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<p>I think there's a TODO missing here. Also, in the exercise, it would be good to mention what arguments need to be passed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>